cardano-node-emulator/test/Cardano/Node/Emulator/golden/logs - payToAddress.txt

```diff
@@ -30,7 +30,8 @@
            redeemers:
 [INFO] Signing tx: 4f54ed37832dae6b107c2a0ec1df8ae2718fe1419d0efd414dd124d2fb4d689b
 [INFO] Submitting tx: 4f54ed37832dae6b107c2a0ec1df8ae2718fe1419d0efd414dd124d2fb4d689b
-[INFO] TxnValidate 4f54ed37832dae6b107c2a0ec1df8ae2718fe1419d0efd414dd124d2fb4d689b [  ]
+[INFO] TxnValidation 4f54ed37832dae6b107c2a0ec1df8ae2718fe1419d0efd414dd124d2fb4d689b
+Validation success
```

Comment:
Should we use the same logging mechasim as the other logs? Something like `[INFO] Validation success 4f54ed37832dae6b107c2a0ec1df8ae2718fe1419d0efd414dd124d2fb4d689b`?

Reply:
Or integrate it with the `TxnValidation` log line

Reply:
It's just one message with a line break.

Reply:
I'll fix the log message rendering so that new lines are rendered indented, but since I have already 2 branches based on this one I won't do that in this PR.

Reply:
:+1: I'll merge then